test_runner/src/main/kotlin/ftl/reports/util/ReportManager.kt

```diff
@@ -159,22 +155,16 @@ object ReportManager {
         }
     }
 
-    private fun createAndUploadPerformanceMetricsForAndroid(
+    // TODO Adam
+/*    private fun createAndUploadPerformanceMetricsForAndroid(
```

Comment:
whats going on here ? :)

Reply:
It's currently fixed, I will remove this TODO